### DISCUSSION TITLE
Run without GitHub action

### DISCUSSION THREAD

**Discussion by joaopedrodcf** [LironEr/bundlemon#183] [Q&A] [answered]
Hey thanks for this project 🥇

I wanted to know if its possible to run the cmd but not using the github action ? For example I'm my own github workflow doing `npm run build`.

In my package.json: `build: next build && bundlemon`

But without adding the github action my pipeline fails running the build because missing project id

![image](https://github.com/LironEr/bundlemon/assets/11476152/cf8b4700-48df-47fb-a59a-c125d3978d46)

**Comment by LironEr** [comment, marked as answer]
Yes, actually the BundleMon GitHub Action is pretty new, but you can still use it without the action.

[This](https://github.com/LironEr/bundlemon/tree/5806a5ffbefaa12614fa72694c137b484ba3fe18#github-action-example--forks-support) is an example workflow without BundleMon GitHub Action.

Anyway, if you are using GitHub actions workflows you need to install BundleMon GitHub App, as described [here](https://github.com/LironEr/bundlemon/tree/5806a5ffbefaa12614fa72694c137b484ba3fe18#bundlemon-project)

If you don't want to install the App, you can also create create a project and use API key.
- [Create new project](https://app.bundlemon.dev/create-project) and copy the project ID and API key
- Add the ID to `BUNDLEMON_PROJECT_ID` and the API key to `BUNDLEMON_PROJECT_APIKEY` environment variables in your CI

If you want GitHub integration without BundleMon GitHub App, you will need to [create GitHub access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) without any scopes, the token owner must have write permission to the repo you would want to post outputs to.

Add the token to `BUNDLEMON_GITHUB_TOKEN` environment variable in your CI.

**Reply by joaopedrodcf** [reply]
Thank you for such a fast reply 🥇
